client-dotnet/src/EvoMaster.Instrumentation/AdditionalInfo.cs

```diff
@@ -1,6 +1,209 @@
 ﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading;
+using EvoMaster.Client.Util;
+using EvoMaster.Instrumentation.StaticState;
+using EvoMaster.Instrumentation_Shared;
+using EvoMaster.Instrumentation_Shared.Collections;
 
-namespace EvoMaster.Instrumentation {
+namespace EvoMaster.Instrumentation
+{
     [Serializable]
-    public class AdditionalInfo { }
+    public class AdditionalInfo
```

Comment:
was there any reason for this to be Serializable?

Reply:
Actually not. The only reason was that the counterpart file in java client is serializable

Reply:
true. that was due to handling of ExternalDriver, which is not needed for C#